README.md

```diff
@@ -1,63 +1,91 @@
-## randomness-solidity
+# Randomness-Solidity
```

Comment:
would suggest either `Randomness Solidity` or `randomness-solidity` - the mixed version looks strange